dotnet/src/Planners/Planners.Core/Handlebars/HandlebarsTemplateEngineExtensions.cs

```diff
@@ -378,14 +358,20 @@ private static double CastToNumber(object? number)
      * Handlebar argument type is an object (this covers complex types).
      * Function parameter is a generic type.
      */
-    private static bool IsExpectedParameterType(string functionViewType, string handlebarArgumentType, object handlebarArgValue)
+    private static bool IsExpectedParameterType(SKParameterMetadata parameterType, object argument)
```

Comment:
I made a best guess on this code, but I'm not sure if/how the previous code was actually correct, and it doesn't look like tests are exercising any of this.

Reply:
I will follow up on test coverage for the Handlebars planner